beets/importer.py

```diff
@@ -1327,7 +1327,24 @@ def resolve_duplicates(session, task):
             log.debug(u'found duplicates: {}'.format(
                 [o.id for o in found_duplicates]
             ))
-            session.resolve_duplicate(task, found_duplicates)
+
+            # Get the default action to follow from config.
+            duplicate_action = config['import']['duplicate_action'].get()
```

Comment:
In the previous PR, you (cleverly) used `as_choice` to validate the configuration, which should be one of `skip`, `keep`, `remove`, and `ask`. Can we restore that?